crates/uv/src/commands/pip_freeze.rs

```diff
@@ -28,12 +28,12 @@ pub(crate) fn pip_freeze(
     let venv = if let Some(python) = python {
         PythonEnvironment::from_requested_python(python, &platform, cache)?
     } else if system {
-        PythonEnvironment::from_default_python(&platform, cache)?
+        PythonEnvironment::from_default_python(&platform, cache, true)?
```

Comment:
You could consider passing `system` here and on line 36. It might actually be clearer IMO, since we don't have the naked bool.